tests/unit/src/util/file.ts

```diff
@@ -0,0 +1,77 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import loadModule, { cleanupModuleMocks } from '../../../_support/loadModule';
+import { stub, SinonStub } from 'sinon';
+
+let module: any;
+let existsSyncStub: SinonStub;
+let mkdtempSyncStub: SinonStub;
+let syncStub: SinonStub;
+let joinStub: SinonStub;
+
+registerSuite({
+	name: 'util/file',
+
+	before() {
+		existsSyncStub = stub();
+		mkdtempSyncStub = stub();
+		syncStub = stub();
+		joinStub = stub();
+	},
+
+	after() {
+		cleanupModuleMocks();
+	},
+
+	beforeEach() {
+		module = loadModule('src/util/file', {
+			'fs': {
+				existsSync: existsSyncStub,
+				mkdtempSync: mkdtempSyncStub
+			},
+			'mkdirp': {
+				sync: syncStub
```

Comment:
let's rename this mkdirpSyncStub so it's easier to understand what this stub does